docs/reference/mapping/runtime.asciidoc

```diff
@@ -44,13 +47,13 @@ smaller and you can more quickly process logs without having to index them.
 
 Runtime fields are especially useful in the following contexts:
 
-* Adding fields to documents that are already indexed without having to reindex
+* Add fields to documents that are already indexed without having to reindex
 data
-* Immediately begin working on a new data stream without fully understanding
-the data it contains
-* Shadowing an indexed field with a runtime field to fix a mistake after
-indexing documents
-* Defining fields that are only relevant for a particular context (such as a
+* Start work on a new data stream without fully understanding the data it
+contains
+* Override the value of an indexed field with a runtime field to fix a mistake
+after indexing documents
+* Define fields that are only relevant for a particular context (such as a
```

Comment:
doesn't this repeat what we already state above?

Reply:
Pretty much. I think we can remove this entire section now since it's restating what is now in the introduction.